internal/persistence/ent/client/sip.go

```diff
@@ -135,7 +174,7 @@ func (c *client) DeleteSIP(ctx context.Context, id int) error {
 
 // ReadSIP returns the SIP identified by id.
 func (c *client) ReadSIP(ctx context.Context, id uuid.UUID) (*datatypes.SIP, error) {
-	s, err := c.ent.SIP.Query().Where(sip.UUID(id)).Only(ctx)
+	s, err := c.ent.SIP.Query().Where(sip.UUID(id)).WithUser().Only(ctx)
```

Comment:
If we only need the UUID, inside `WithUser()` we can limit the fields returned by the query as an small improvement.